sdc/tests/test_dataframe.py

```diff
@@ -1143,15 +1143,46 @@ def test_impl(A):
         hpat_func = self.jit(test_impl)
         pd.testing.assert_frame_equal(hpat_func(df), test_impl(df2))
 
-    @skip_numba_jit
-    def test_df_reset_index1(self):
+    def test_df_reset_index_drop_True(self):
         def test_impl(df):
             return df.reset_index(drop=True)
 
         df = pd.DataFrame({'A': [1.0, 2.0, np.nan, 1.0]})
         hpat_func = self.jit(test_impl)
         pd.testing.assert_frame_equal(hpat_func(df), test_impl(df))
 
+    def test_df_reset_index_drop_False(self):
+        def test_impl(df):
+            return df.reset_index(drop=False)
+
+        df = pd.DataFrame({'A': [1.0, 2.0, np.nan, 1.0]})
+        hpat_func = self.jit(test_impl)
+
+        pd.testing.assert_frame_equal(hpat_func(df), test_impl(df))
+
+    def test_df_reset_index_df_different_data(self):
```

Comment:
What's the point of this test, to test that impl works for multiple columns as it does for single one? If so it's better to update previous tests (test_df_reset_index_drop_False, etc), so that DF in them has more than one column. Testing various data types/runtime values is redundant in my opinion, as method impl does not depend on them at all.